Description: node is called nodejs in Debian
 Use the proper binary name.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2018-09-20

---

--- thrift-0.9.3.orig/lib/nodejs/test/testAll.sh
+++ thrift-0.9.3/lib/nodejs/test/testAll.sh
@@ -40,7 +40,7 @@ testServer()
     ${ISTANBUL} cover ${DIR}/server.js --dir ${REPORT_PREFIX}${COUNT} --handle-sigint -- --type $1 -p $2 -t $3 $4 &
     COUNT=$((COUNT+1))
   else
-    node ${DIR}/server.js --type $1 -p $2 -t $3 $4 &
+    nodejs ${DIR}/server.js --type $1 -p $2 -t $3 $4 &
   fi
   SERVERPID=$!
   sleep 0.1
@@ -48,7 +48,7 @@ testServer()
     ${ISTANBUL} cover ${DIR}/client.js --dir ${REPORT_PREFIX}${COUNT} -- --type $1 -p $2 -t $3 $4 || RET=1
     COUNT=$((COUNT+1))
   else
-    node ${DIR}/client.js --type $1 -p $2 -t $3 $4 || RET=1
+    nodejs ${DIR}/client.js --type $1 -p $2 -t $3 $4 || RET=1
   fi
   kill -2 $SERVERPID || RET=1
   wait $SERVERPID
@@ -59,7 +59,7 @@ testBrowser()
 {
   echo "   Testing browser client with http server with json protocol and buffered transport";
   RET=0
-  node ${DIR}/server.js --type http -p json -t buffered &
+  nodejs ${DIR}/server.js --type http -p json -t buffered &
   SERVERPID=$!
   sleep 1
   ${RUNBROWSER} ${DIR}/browser_client.js --phantom || RET=1
@@ -76,8 +76,8 @@ ${DIR}/../../../compiler/cpp/thrift -o $
 
 #unit tests
 
-node ${DIR}/binary.test.js || TESTOK=1
-node ${DIR}/deep-constructor.test.js || TESTOK=1
+nodejs ${DIR}/binary.test.js || TESTOK=1
+nodejs ${DIR}/deep-constructor.test.js || TESTOK=1
 
 #integration tests
 
